### ISSUE TITLE
Add a lofting method to the Python API Modeling module

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/SimVascular#747]
The Python API Modeling module does not have a `loft_surface()` method. There were some issues with the SV solid modeling interface that prevented me from implementing this: it was not supported for all modelers. 

I will also add a capping method.

**Comment by ktbolt** [comment]
The SV `cvSolidModel` interface is a mess, half of the methods are not implemented for the interface and just return an error, for example `cvOCCTSolidModel`

```
  int MakeApproxCurveLoop( cvPolyData *pd, double tol, int closed ) {return SV_ERROR; }
```

I will just implement a loft method for the Parasolid modeler for now, will take a lot of work to get the other modelers working.

**Comment by ktbolt** [comment]
I've add a couple of new methods to the API `modeling` module `Modeler` class for Parasolid to create interpolated curves (needed for lofting) and creating a lofted surface from them
```
# Create a Parasolid modeler.
modeler = sv.modeling.Modeler(sv.modeling.Kernel.PARASOLID)

# Align contours and create curves from them. 
for cid in range(start_cid,end_cid):
    cont_ipd = get_profile_contour(gr, renderer, contours, cid, num_profile_points)
    if cid == start_cid:
        cont_align = cont_ipd
    else:
        cont_align = sv.geometry.align_profile(last_cont_align, cont_ipd, use_distance)
    curve = modeler.interpolate_curve(cont_align)
    curve_list.append(curve)
    last_cont_align = cont_align

# Create a lofted surface. 
loft_surf = modeler.loft(curve_list=curve_list)
```
![Screen Shot 2020-09-25 at 7 51 04 PM](https://user-images.githubusercontent.com/23109458/94328690-4efbdf80-ff69-11ea-9b01-6b44246be889.png)

@suhaasa The SV lofting method `MakeLoftedSurf()` takes several undocumented parameters: continuity, partype, w1, w2, w3, smoothing. I will merge this code and build a new installer when I can figure out what these guys are used for.

**Comment by ktbolt** [comment]
Looking at the `cvParasolidSolidModel::MakeLoftedSurf()` method I see that none of the `continuity, partype, w1, w2, w3, smoothing` parameters passed to this method are used! 

And of course this 
```
// ----------------
// PsdUtils_MakeCap
// ----------------
// Adapted from D. Parker.

int PsdUtils_MakeCap( cvPolyData *profile, PsdListT **result )
```

worries me a bit ...

**Comment by suhaasa** [comment]
Thanks Dave - let me know when you have an installer ready for ubuntu 16! I'll try it out and see if it'll work!

**Comment by ktbolt** [comment]
I needed to add a flag to not cap to the`MakeLoftedSurf()` method was always because Parasolid  was lofting and capping surfaces there. I also exposed the SV modeling `CapSurfToSolid()` method to explicitly cap lofted surfaces. Without code comments or GitHub Issues it is impossible to understand why the `CapSurfToSolid()` method was not exposed. 

Lofting and capping now seems to work for Parasolid and Open Cascade. I've added checks that throw an error for PolyData modeler. At some point I will add lofting and capping to the PolyData modeling.
